# linux port and AX-12A baud rate
usb_port: "/dev/ttyUSB0"
dxl_baud_rate: 1000000

# control table addresses
addr_torque_enable: 24
addr_goal_position: 30
addr_present_position: 36

# AX-12A min max
dxl_minimum_position_value: 0
dxl_maximum_position_value: 1023

# Protocol version
protocol_version: 1.0